ui/src/wide_gauge_stat.tsx

```diff
@@ -76,8 +73,19 @@ interface State {
 
 export const
   View = bond(({ name, state: s, changed }: Model<State>) => {
+    const
+      valueNonDigitCharCount = substitute(s.value, s.data).replace(/[0-9]/g, '').length,
+      auxValueNonDigitCharCount = substitute(s.aux_value, s.data).replace(/[0-9]/g, '').length
     const render = () => {
-      const data = unpack<Rec>(s.data)
+      const
+        data = unpack<Rec>(s.data),
+        /*
+         * Numbers 14 and 9 are representing maximum width of the single character.
```

Comment:
use properly named constants instead of this comment. Something like `MAX_AUX_WIDTH` etc.

Reply:
Yeah, I was deciding about this option as well. I'll do it!